test/stress/logs/agent_config.json

```diff
@@ -0,0 +1,53 @@
+{
+    "agent": {
+      "debug": true
+    },
+    "metrics": {
+      "namespace": "CloudWatchAgentStress",
+      "metrics_collected": {
+        "net": {
+          "resources": [
+            "eth0"
+          ],
+          "measurement": [
+            "bytes_sent",
+            "packets_sent"
+          ],
+          "metrics_collection_interval": 1
+        },
+        "procstat": [
+          {
+            "exe": "cloudwatch-agent",
+            "measurement": [
+              "cpu_usage",
+              "memory_rss",
+              "memory_swap",
+              "memory_vms",
+              "memory_data",
+              "num_fds"
+            ],
+            "metrics_collection_interval": 1
+          }
+        ]
+      },
+      "append_dimensions": {
+        "InstanceId": "${aws:InstanceId}"
+      },
+      "force_flush_interval": 10
+    },
```

Comment:
If we are doing the stress test for `logs`, why do we need to have `metrics` config here? Or are we checking with the `logs` configured, how's CWAgent processing metrics at the same time?

Reply:
Since the metrics being submitted by CWAgent are used by the stress to validate if the metrics are within bound (e.g cpu_usage). So yes we would need a metrics section here to get all the [required metrics by the stress test ](https://github.com/aws/amazon-cloudwatch-agent-test/blob/37e28a3e8f676415113b43b443ff9102a2f46912/validator/validators/stress/stress_validator.go#L31-L40). Even you could argue it would corrupt the CWAgent resources; however, we have determine it would be a small amount so we can safely ignore it.